src/manager/frontend/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "ragapp-admin-dashboard",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "export ENVIRONMENT=dev && next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@radix-ui/react-avatar": "^1.1.0",
+    "@radix-ui/react-icons": "^1.3.0",
+    "@radix-ui/react-slot": "^1.1.0",
+    "@radix-ui/react-tooltip": "^1.1.2",
+    "class-variance-authority": "^0.7.0",
+    "clsx": "^2.1.1",
+    "dotenv": "^16.4.5",
+    "lucide-react": "^0.414.0",
+    "next": "14.2.5",
+    "react": "^18",
+    "react-dom": "^18",
+    "react-query": "^3.39.3",
+    "tailwind-merge": "^2.4.0",
+    "tailwindcss-animate": "^1.0.7",
+    "react-icons": "^5.2.1"
```

Comment:
TBD: might just use lucide for icons, e.g. https://lucide.dev/icons/bot for agent

Reply:
or alternatively, as `react-icons` includes lucide's icons (see https://react-icons.github.io/react-icons/icons/fi/), just use lucide

Reply:
i mean react-icons